plugin-server/src/utils/db/hub.ts

```diff
@@ -151,6 +153,10 @@ export async function createHub(
         serverConfig.EXTERNAL_REQUEST_TIMEOUT_MS
     )
 
+    const actionManager = new ActionManager(postgres, serverConfig)
+    await actionManager.start() // TODO: Do we want this to happen here?
+    const actionMatcher = new ActionMatcher(postgres, actionManager)
```

Comment:
we probably don't want this here - this is only relevant to webhooks and export plugins, not all the other plugin-server instances

Reply:
maybe we initialize these to undefined and init them in `startAsyncWebhooksHandlerConsumer` and `startAsyncOnEventHandlerConsumer`?